nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Metrics data to ServiceControl
+summary: Reporting metrics to ServiceControl.
+reviewed: 2017-09-26
+component: MetricsServiceControl
+related:
+ - nservicebus/operations
+related:
+ - samples/logging/metrics
+---
+
+
+This component enables sending monitoring data to ServiceControl.Monitoring instance.
+
+
+## Enabling
+
+It can be enabled via:
+
+snippet: SendMetricDataToServiceControl
+
+
+#### Service Control Metrics Address:
```

Comment:
Do we need semicolons here and at the end of next section?

Reply:
I'll remove them